test/acceptance/integration/sth/notifications.feature

```diff
@@ -0,0 +1,241 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of Short Term Historic (FI-WARE project).
+#
+# iot-sth is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+# iot-sth is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with iot-sth. If not, see
+# http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact:
+# iot_support at tid.es
+#
+__author__ = 'Iván Arias León (ivan.ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+Feature: Store in mongo new notifications from context broker in raw and aggregated format
+  As a sth user
+  I want to be able to Store in mongo new notifications from context broker in raw and aggregated format
+  so that they become more functional and useful
+
+  @happy_path
+  Scenario: stored new notifications in mongo from context broker with metadata
+    Given copy properties.json file from "epg_properties.json" and sudo local "false"
```

Comment:
Consider do a "background" step (to do always the same in this feature), or do this "copy" in the "before_all" in the terrain if its going to be always the same (in all features)